packages/openops/src/lib/ai/providers.ts

```diff
@@ -72,11 +72,19 @@ export const getAiProviderLanguageModel = async (aiConfig: {
   providerSettings?: Record<string, unknown> | null;
 }): Promise<LanguageModelV1> => {
   const aiProvider = getAiProvider(aiConfig.provider);
-
+  const sanitizedSettings = sanitizeProviderSettings(aiConfig.providerSettings);
+  const mappedProviderSettings: Record<string, unknown> = {
```

Comment:
we could also just rename it in the frontend, to be "baseURL" instead i dont mind either way

Reply:
that would be much cleaner imo